core/dbt/include/global_project/macros/python_model/python.sql

```diff
@@ -1,9 +1,19 @@
+{% macro resolve_model_name(input_model_name) %}
+
+  {{ return(adapter.dispatch('resolve_model_name', 'dbt')(input_model_name)) }}
```

Comment:
I think this is actually the first time I see dispatch being implemented since I join